rustup-toolchain

```diff
@@ -21,22 +21,32 @@ if ! which rustup-toolchain-install-master >/dev/null; then
     exit 1
 fi
 
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
```

Comment:
I don't know what this rune does, but anyway if the entire logic moves to `miri` it can just use the `MIRIDIR` variable that exists there.